data/transaction/transaction.go

```diff
@@ -88,7 +88,7 @@ func TransactionGoToCapn(seg *capn.Segment, src *Transaction) capnp.TransactionC
 	dest.SetSndAddr(src.SndAddr)
 	dest.SetGasPrice(src.GasPrice)
 	dest.SetGasLimit(src.GasLimit)
-	dest.SetData(src.Data)
+	dest.SetData([]byte(src.Data))
```

Comment:
I would also change the schema instead of converting the string to `Data` format

Reply:
Done